robottelo/common/constants.py

```diff
@@ -169,7 +169,9 @@
     ('rhel', 'rhva6', 'rhva6S', 'repo_ver', '6Server')
 ]
 
+#: Name (not label!) of the default organization.
```

Comment:
This will not needed if the constant was named `DEFAULT_ORG_NAME`. Maybe this worth being refactored in the future.